lib/ansible/modules/cloud/vmware/vmware_guest_attribute.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/python
```

Comment:
i'd rename it to `vmware_guest_custom_attribute` (or `vmware_guest_custom_attributes`, see further). It better reflects what attributes it is about (there are a lot of attributes, someone with less experience in VMware might see 'attributes' as the normal name or even hw configuration...